cmd/main.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/Sirupsen/logrus"
+	config "github.com/openebs/zfs-localpv/pkg/config/v1alpha1"
+	service "github.com/openebs/zfs-localpv/pkg/service/v1alpha1"
+	"github.com/openebs/zfs-localpv/pkg/version"
+	"github.com/spf13/cobra"
+)
+
+func main() {
+	_ = flag.CommandLine.Parse([]string{})
+	var config = config.Default()
+
+	cmd := &cobra.Command{
+		Use:   "openebs-csi-driver",
+		Short: "openebs-csi-driver",
```

Comment:
I still did not understand the use of this command.
Can we have better short & long descriptions

Reply:
add details to the cmd.

Reply:
added details.